locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -1391,21 +1396,21 @@ msgstr ""
 #. http://i.imgur.com/n6W3M3I.png
 msgctxt "settings"
 msgid "Custom"
-msgstr ""
+msgstr "Kesane"
 
 #. Custom --> 'customized'
 #. A setting that *you* choose and is not a default.
 msgctxt "settingsvalue"
 msgid "Custom"
-msgstr ""
+msgstr "Kesane"
 
 msgctxt "settings"
 msgid "Custom %s"
-msgstr ""
+msgstr "Kesane%s"
 
 msgctxt "settingsvalue"
 msgid "Custom %s"
-msgstr ""
+msgstr "Kesane%s"
```

Comment:
Shouldn't we have a space before the placeholder here?

```suggestion
msgstr "Kesane %s"
```

Reply:
Yeah my bad, should be there a space